hail_scripts/reference_data/clinvar.py

```diff
@@ -55,6 +56,20 @@
 )
 
 
+def safely_move_to_gcs(tmp_file_name, gcs_tmp_file_name):
+    try:
+        subprocess.run(
+            [  # noqa: S603, S607
+                'gsutil',
+                'cp',
```

Comment:
Curious if we should use the multithread feature of `gsutil -m cp...`. Also not sure if we plan to use this outside of vcfs, would it hurt to through in the recursive option too? Or is that adding confusion? `gsutil -m cp -r`?

Reply:
Yeah... I'm unsure what to do here.  I ran into several issues here and this was kinda my best stab at getting something working.

Our old approach had a line that added the tmp file to `hdfs`, which is the distributed filesystem shared by all the dataproc nodes and I had removed it several PRs ago because it blocked local testing.  After running into dataproc failures, I re-added that line... which worked for a while... and then _stopped_ working once we scaled the cluster because `hdfs` stopped being able to find the file.  Just using `gcs` with this command worked fine.